config/environment.js

```diff
@@ -97,6 +97,15 @@ module.exports = function(environment) {
     enabled: false
   };
 
+  ENV['mapbox-gl'] = {
+  	 accessToken: 'pk.eyJ1Ijoia3VzaHRoZWR1ZGUiLCJhIjoiY2s0bmczdXhlMzJidDNlbXdpY3F2enp3dSJ9.fZMstKnXl5ySzxzMajg3vQ',
+      map: {
+    style: 'mapbox://styles/mapbox/basic-v9',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 6 spaces but found 4.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)